src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/GEOS_MetforceGridComp.F90

```diff
@@ -677,13 +678,10 @@ subroutine Initialize(gc, import, export, clock, rc)
     VERIFY_(STATUS)
     ensemble_forcing = (trim(ENS_FORCING_STR) == 'YES') 
     if (ensemble_forcing .and. NUM_ENSEMBLE > 1) then
-      id_string = ""
-      call MAPL_GetResource ( MAPL, ens_id_width, Label="ENS_ID_WIDTH:",      DEFAULT=0,       RC=STATUS)
+      ! note: comp_name ends in "_eXXXX"; for GEOS ADAS forcing, extract hard-coded 3-digit ens id string
       k = len(trim(comp_name))
-      id_string = comp_name(k-ens_id_width+1:k)
-      k = len(trim(id_string)) 
-      ! hard coded 3 character for forcing
-      call ESMF_CFIOStrTemplate(ens_forcing_path, trim(adjustl(mf%Path)),'GRADS', xid = trim(id_string(k-2:k)), stat=status)
+      ensid_string3 = comp_name(k-2:k)
```

Comment:
This would fail if a user specifies only 2 digits for ens id in LDAS.rc.  Should we add check that ENS_ID_WIDTH must be at least 3?  (not counting the extra 2 char for "_e")